sdk/containerregistry/test-resources-post.ps1

```diff
@@ -0,0 +1,12 @@
+param (
+    [hashtable] $DeploymentOutputs,
```

Comment:
Is the post operation consistent across all languages? I do not see it with Java?

Reply:
@pallavit Thanks for reviewing! I just copied them from .NET to get my live test pipeline passing. I don't have any live tests yet so I don't know how this works.

Reply:
@annelo-msft can you help us understand this a little bit more.

---
In reply to: [596461424](https://github.com/Azure/azure-sdk-for-js/pull/14320#discussion_r596461424) [](ancestors = 596461424)

Reply:
Sure!

In .NET, our New-TestResources.ps1 file invokes the test-resources-post.ps1 script after it's created the test resources.  You can see where it passes `DeploymentOutputs` to the script [here](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/common/TestResources/New-TestResources.ps1#L513).

The values in `DeploymentOutputs` are defined in the [test-resources.json](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/containerregistry/test-resources.json) file, and by New-TestResources.ps1

Does that answer your question?